code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -318,6 +282,123 @@
 	busy = SPIDER_IDLE
 	stop_automated_movement = FALSE
 
+// Midwives are upgraded nurses. They can web quickly and are stronger than regular nurses, but they're a bit slower.
+/mob/living/simple_animal/hostile/poison/giant_spider/nurse/midwife
+	name = "midwife"
+	desc = "Furry and black, it makes you shudder to look at it. This one has scintillating green eyes."
+	icon_state = "midwife"
+	icon_living = "midwife"
+	icon_dead = "midwife_dead"
+	maxHealth = 80
+	health = 80
+	speed = 2
+	web_speed = 0.15 // Easily able to web
+	// Allows the spider to use spider comms
+	var/datum/action/innate/spider/comm/letmetalkpls
+
+/mob/living/simple_animal/hostile/poison/giant_spider/nurse/midwife/Initialize(mapload)
+	. = ..()
+	letmetalkpls = new
+	letmetalkpls.Grant(src)
+
+/mob/living/simple_animal/hostile/poison/giant_spider/nurse/midwife/Destroy()
+	QDEL_NULL(letmetalkpls)
+	return ..()
+
+// Hunters have a decent amount of poison and have decent general stats, making them offensive spiders. They're a bit squishier
+// than regular spiders, though
+/mob/living/simple_animal/hostile/poison/giant_spider/hunter
+	name = "hunter"
+	desc = "Furry and black, it makes you shudder to look at it. This one has sparkling purple eyes."
+	icon_state = "hunter"
+	icon_living = "hunter"
+	icon_dead = "hunter_dead"
+	maxHealth = 80
+	health = 80
+	melee_damage = 20
+	poison_per_bite = 5
+	move_to_delay = 3
+	speed = 0
+
+// vipers are the upgraded variant of the hunter, able to move quickly and inject a lot of venom.
+/mob/living/simple_animal/hostile/poison/giant_spider/hunter/viper
+	name = "viper"
+	desc = "Furry and black, it makes you shudder to look at it. This one has effervescent purple eyes."
+	icon_state = "viper"
+	icon_living = "viper"
+	icon_dead = "viper_dead"
+	maxHealth = 50
+	health = 50
+	melee_damage = 1
+	poison_per_bite = 12
+	move_to_delay = 2
+	poison_type = /datum/reagent/toxin/venom
+	gold_core_spawnable = NO_SPAWN
+
+//tarantulas are really tanky, but slower than normal spiders. They can also break stuff much easier, and can do more damage.
+/mob/living/simple_animal/hostile/poison/giant_spider/tarantula
+	name = "tarantula"
+	desc = "Furry and black, it makes you shudder to look at it. This one has abyssal red eyes."
+	icon_state = "tarantula"
+	icon_living = "tarantula"
+	icon_dead = "tarantula_dead"
+	maxHealth = 300
+	health = 300
+	melee_damage = 35
```

Comment:
```suggestion
	maxHealth = 160
	health = 160
	melee_damage = 25
	poison_per_bite = 0
```

As the primary defenders they deserve to have more HP, but 300 is absurdly high for a mob that is intended by design to protect and move in close proximity with others capable of healing it.